Vagrantfile

```diff
@@ -135,4 +135,18 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     agent.vm.provision :shell, :inline => "puppet apply /vagrant/tests/sensu-agent.pp"
     agent.vm.provision :shell, :inline => "facter --custom-dir=/vagrant/lib/facter sensu_agent"
   end
+
+  config.vm.define "win2012r2-agent", autostart: false do |agent|
```

Comment:
should also have windows 2016 since we test for that with appveyor

Reply:
I ran into an issue with sensuclassic module where it was really hard to find a reliable image, we eventually removed 2016 since it was hit or miss if the images would work.

Reply:
Managed to get a 2016 box working.